Resources/Prototypes/Loadouts/loadout_groups.yml

```diff
@@ -35,6 +35,16 @@
   - GlassesJamjar
   - GlassesJensen
 
+- type: loadoutGroup
+  id: Survival
+  name: loadout-group-survival-basic
+  minLimit: 2
+  maxLimit: 2
```

Comment:
why 2?

Reply:
There are two loadout options: oxygen and nitrogen. MinLimit is used to make sure the character has a survival box without the player having selected one, but MinLimit does not test the validity of the picks it tries to force. Thus, if the first option is unavailable due to species, MinLimit: 1 will try and fail to add that to the character, ignoring the second option and making the player spawn without a survival box. So for species that can only use the second listed option to work with 0 setup from the player, both possible loadout choices must be forced active.

Reply:
Or uh, the short answer is: the species-check will block the other one so it will not actually be given to the character